level0/cachebw/Makefile

```diff
@@ -29,11 +29,12 @@
 
 CC = gcc
 #CFLAGS = -O2 -fopenmp -mcpu=power8 -mvsx -mdirect-move -mpower8-fusion -mpower8-vector -fstrict-aliasing
-CFLAGS = -O2 -fopenmp -march=armv8-a -fstrict-aliasing
+#CFLAGS = -O2 -fopenmp -march=armv8-a -fstrict-aliasing
+CFLAGS = -O2 -fopenmp -march=rv64imfdv -fstrict-aliasing
 #CFLAGS = -O2 -fopenmp -mavx2 -fstrict-aliasing
 #CFLAGS = -O2 -fopenmp -msse3 -fstrict-aliasing
 #CFLAGS = -O2 -fopenmp -msse3 -fstrict-aliasing
-CFLAGS = -O2 -msse -fstrict-aliasing
+#CFLAGS = -O2 -msse -fstrict-aliasing
```

Comment:
Please just added the RV64 options and leave the x86 option override as default
(we can work after initial merge to add a TARGET as used in the other benchmarks)